src/pages/ConditionsList/index.tsx

```diff
@@ -329,6 +334,35 @@ export const ConditionsList: React.FC = () => {
     [error]
   )
 
+  React.useEffect(() => {
+    if (
+      textToSearch !== '' ||
+      selectedOracleValue !== OracleFilterOptions.All ||
+      selectedOracleFilter.length !== 0 ||
+      selectedConditionTypeValue !== ConditionTypeAll.all ||
+      selectedConditionTypeFilter ||
+      selectedStatus !== StatusOptions.All ||
+      selectedMinOutcomes ||
+      selectedMaxOutcomes ||
+      selectedToCreationDate ||
+      selectedFromCreationDate
```

Comment:
Maybe here an useMemo(... with && for each condition) as 'areFiltersCleared' or something like that. An do `if(!areFiltersCleared)`.

Reply:
This change will memoize the update of some variables, and the use of `areFiltersCleared` will not trigger the reset pagination